pkg/api/api.go

```diff
@@ -260,6 +263,8 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Get("/frontend/settings/", hs.GetFrontendSettings)
 		apiRoute.Any("/datasources/proxy/:id/*", reqSignedIn, hs.ProxyDataSourceRequest)
 		apiRoute.Any("/datasources/proxy/:id", reqSignedIn, hs.ProxyDataSourceRequest)
+		apiRoute.Any("/datasources/:id/resources", Wrap(hs.CallDatasourceResource))
+		apiRoute.Any("/datasources/:id/resources/*", Wrap(hs.CallDatasourceResource))
```

Comment:
We also need:
apiRoute.Get("/datasources/:id/health", Wrap(hs.CheckHealth))

This would need to get passed jsondata and secureJsonData and be able to check that the connections all work

Reply:
This is on the todo, but will get this merged first.